src/classes/EscalationTrigger_Test.cls

```diff
@@ -186,4 +186,58 @@ public class EscalationTrigger_Test {
 		System.assertEquals(null, testEscalation.rh_escal__PrimaryCase__c, 'The primary case of escalation should be null.');
 		System.assertEquals(null, testEscalation.rh_escal__Account__c, 'The account is not set correctly.');
 	}
+
+	////////////////////////////////////////////////////////////////////////////////////
+	//                   createEscalationCaseRecordOnEscalationInsert                 //
+	////////////////////////////////////////////////////////////////////////////////////
+
+	static testMethod void createEscalationCaseRecord_primaryCaseOnEscalationNotNull_valid() {
+		Account testAccount = TestUtils.getAccount();
+		insert testAccount;
+
+		Case testCase = TestUtils.getCase(testAccount);
+		insert testCase;
+
+		rh_escal__Escalation__c testEscalation = TestUtils.getEscalation();
+		TestUtils.setPrimaryCaseOnEscalation(testEscalation, testCase);
+
+		Test.startTest();
+
+		insert testEscalation;
+
+		Test.stopTest();
+
+		testEscalation = TestUtils.fetchEscalation(testEscalation);
+
+		rh_escal__EscalationCase__c escalCase = TestUtils.getEscalationCase(testEscalation, testCase);
```

Comment:
This doesn't do what we want it to do.  We need to add a new method to `TestUtils` that fetches all escalation cases and then we can do our asserts on that

```java
/**
* Fetches all escalation cases for a given escalation
*
* @param escalation The escalation to get it's cases
* @return The escalation cases
*/
global static List<rh_escal__EscalationCase__c> fetchEscalationCases(rh_escal__Escalation escalation) {
    return [
        select rh_escal__Case__c,
            rh_escal__Escalation__c,
            rh_escal__Primary__c
        from rh_escal__EscalationCase__c
        where rh_escal__Escalation__c = :escalation.Id
    ];
}
```

Then update line 212 to be

```java
List<rh_escal__EscalationCase__c escalCases = TestUtils.fetchEscalationCases(testEscalation);
System.assertEquals(1, escalCases.size(), 'Did not get the expected number of cases back');
rh_escal__EscalationCase__c escalCase = escalCases.get(0);
```